### PR TITLE
AMPscript extensions checker

### PR DESCRIPTION
**Pull request by FiB3**
The goal of this PR is to help users regarding the issues #13 & #16 by notifying them about the currently incompatible extensions.

### CODE REVIEW

File: src/vscode/src/checks.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import * as vscode from 'vscode';
+
+export function extensionsChecks() {
+	const extensionHandlers = {
+		"sergey-agadzhanov.AMPscript": {
+			handler: () => {
+				vscode.window.showInformationMessage(`MCFS [AMPScript] extension is currently not supported by AMPscript Core.`);
```
Comment (lbuesching) [inline src/vscode/src/checks.ts]: Let's update this message to convey what not supported means.

"The extension MCFS [AMPScript] is incompatible with AMPscript Core and will prevent AMPscript Core from debugging AMPscript."

File: src/vscode/src/checks.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+				return false;
+			}
+		},
+		"FiB.beautyAmp": {
+			handler: () => true
```
Comment (lbuesching) [inline src/vscode/src/checks.ts]: Was this to test a positive test case?

**Review by lbuesching — CHANGES_REQUESTED** [review]

**Comment by FiB3** [comment]
Ok, updated the warning message. :)
You are right, the "beautyAmp" was more of an example and test case. I've removed it and replaced it with a comment.

**Comment by lbuesching** [comment]
Backward compatibility checks are fine, no changes from this change.

**Pull request merged**